docs/classes/_bots_versionbot_.versionbot.html

```diff
@@ -380,7 +380,7 @@ <h4 class="tsd-returns-title">Returns <span class="tsd-signature-type">ServiceLi
 						<li class="tsd-description">
 							<aside class="tsd-sources">
 								<ul>
-									<li>Defined in <a href="https://github.com/resin-io-modules/resin-procbots/blob/master/lib/bots/versionbot.ts#L754">bots/versionbot.ts:754</a></li>
+									<li>Defined in <a href="https://github.com/resin-io-modules/resin-procbots/blob/master/lib/bots/versionbot.ts#L803">bots/versionbot.ts:803</a></li>
```

Comment:
I hadn't suppressed them up to now as documenting ProcBots is one of the open issue items, and this shows updates. Although, in this case, it's just line numbers changing as there are very few actual documentation comments changing. It's a tricky one.